src/components/MenuPanel/MenuPanelAppGroup.js

```diff
@@ -29,13 +29,9 @@ class MenuPanelAppGroup extends React.PureComponent {
     } = this.props
     const singleInstance = instances.length === 1
     return (
-      <Motion
-        style={{
-          openProgress: spring(
-            Number(active && (singleInstance || expand)),
-            springConf('fast')
-          ),
-        }}
+      <Spring
+        config={springs.fast}
```

Comment:
Can we try [`springs.swift`](https://github.com/aragon/aragon-ui/blob/300fa250c200c9476bb2776930c7f460332011a7/src/utils/styles/spring.js#L17) instead? `slow` / `normal` / `fast` are only here for compatibility reasons.